### PR TITLE
[sdk] rename cga to gw across repo

### LINKED ISSUE #424: [SDK] rename SDK to GenomeWorks
Description to be updated once final name of repo is decided

### PR DESCRIPTION
**Pull request by tijyojwad**
Final PR to address name change

1. Updates all mentions of cga/CGA to gw/GW (cmake files, docs, macros, etc)
2. Replaes mentions of "Clara Genomics Analysis" with "GenomeWorks"

Closes #424

### CODE REVIEW

File: common/io/CMakeLists.txt [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
-get_property(cga_library_type GLOBAL PROPERTY cga_library_type)
-add_library(${PROJECT_NAME} ${cga_library_type}
+get_property(gw_library_type GLOBAL PROPERTY gw_library_type)
+add_library(${PROJECT_NAME} ${gw_library_type}
         src/fasta_parser.cpp
         src/kseqpp_fasta_parser.cpp)
-target_link_libraries(${PROJECT_NAME} PUBLIC cgabase z)
+target_link_libraries(${PROJECT_NAME} PUBLIC gwbase z)
 
 add_doxygen_source_dir(${CMAKE_CURRENT_SOURCE_DIR}/include/claragenomics/io)
```
Comment (ohadmo) [inline common/io/CMakeLists.txt:19]: Change into `include/claraparabricks/genomeworks/io`

File: CMakeLists.txt [hunk 2]
```diff
@@ -9,17 +9,17 @@
...
-file(READ "${CMAKE_CURRENT_SOURCE_DIR}/VERSION" CGA_VERSION)
-string(STRIP ${CGA_VERSION} CGA_VERSION)
-project(${CGA_PROJECT_NAME})
+set(GW_PROJECT_NAME ClaraGenomicsAnalysis)
+file(READ "${CMAKE_CURRENT_SOURCE_DIR}/VERSION" GW_VERSION)
+string(STRIP ${GW_VERSION} GW_VERSION)
+project(${GW_PROJECT_NAME})
 
 # Process options.
-option(cga_enable_tests "Build ClaraGenomicsAnalysis unit tests" OFF)
```
Comment (ohadmo) [inline CMakeLists.txt:18]: `ClaraGenomicsAnalysis` appears in this file several times

Comment (tijyojwad) [reply CMakeLists.txt:18]: good catch! changed

File: common/io/CMakeLists.txt (hunk 1 above)
Comment (tijyojwad) [reply common/io/CMakeLists.txt]: Good catch, changed

File: CMakeLists.txt (hunk 2 above)
Comment (tijyojwad) [reply CMakeLists.txt:18]: Changed as well

**Pull request merged**